### DISCUSSION TITLE
Playlist empty

### DISCUSSION THREAD

**Discussion by martin-zh** [rern/rAudio#964] [Fixed Issues]
Since one of the last updates my playlist remains empty. The following errors are displayed in the browser console:

Failed to load resource: the server responded with a status of 500 (Internal Server Error): /mpdplaylist.php:1

as well as 

POST http://10.254.254.100/mpdplaylist.php 500 (Internal Server Error): jquery-3.6.0.min.js:2 

Browser is the current Edge Chromium Beta (103.0)

**Comment by Retrofreak** [comment]
I have a similar error since one of the last updates: The library update only works as an "rebuild entire database" update. No more result for "only changed files". Does anyone have any helpful advice? Thanks in advance!

**Comment by rern** [comment]
Please try today's update.

**Reply by Retrofreak** [reply]
Thanks, the "only changed files"-database-update is back from summer holidays

**Comment by martin-zh** [comment]
Unfortunately, the error is still present; the playlist is still empty (even after a library rebuild).
![2022-07-11 07_17_55-rAudio and 3 more pages - Work - Microsoft​ Edge Beta](https://user-images.githubusercontent.com/21334430/178193909-060c2a17-e31d-4a0e-9971-9de523c6a949.png)
![2022-07-11 07_18_33-rAudio and 3 more pages - Work - Microsoft​ Edge Beta](https://user-images.githubusercontent.com/21334430/178193915-9ed21bf9-354d-4a0c-80db-e2a96989992b.png)

**Comment by martin-zh** [comment]
Reinstalling the player has now helped and solved the problem
